_includes/pagination.html

```diff
@@ -4,7 +4,7 @@
   {% else %}
     {% assign currentOffset = 2 %}
   {% endif %}
-  <nav class="u-containProse" role="menu" aria-label="Pagination">
+  <nav class="u-containProse" role="navigation">
```

Comment:
Question: Was `aria-label` removed because the element is understandable without? (Not critiquing, just curious.)